### ISSUE TITLE
Including this mod in modpack

### ISSUE THREAD

**Issue by emmaexe** [henkelmax/gravestone#161]
Hello. I am writing here as I don't have any better solution for contacting you.

I would like to include this mod in a modpack, but I made some modifications to its behaviour. Since there is no license in thid repository and the mod is marked as ARR on modrinth I'd like to request your permission to distribute this modified version in my modpack.

The modpack: https://modrinth.com/modpack/all-the-horrors

The changes: I added a server setting that lets you define a % chance for every item that is on you at the time of death to be dropped instead of being placed in the grave.

**Comment by henkelmax** [comment]
No. You are only allowed to include the original mod in modpacks. Never distribute modified versions!
